doc/manuals/admin/cli.md

```diff
@@ -138,3 +138,5 @@ The list of available options is the following:
 -   **-reqPoolSize**. Size of thread pool for incoming connections. Default value is 0, meaning *no thread pool*.
 -   **-statCounters**, **-statSemWait**, **-statTiming** and **-statNotifQueue**. Enable statistics
     generation. See [statistics documentation](statistics.md).
+-   **-notifErrLogInterval**. Interval to issue warning log about notification errors - in number of errors.
```

Comment:
I would prefer to remove them. Keep things hidden have the risk of forgotting about them (given that they aren't in the documentation). In the worst case, it would be easy to add them back.